csharp/ql/integration-tests/all-platforms/standalone_dependencies/Assemblies.expected

```diff
@@ -0,0 +1,171 @@
+| /microsoft.netcore.app.ref/6.0.13/ref/net6.0/System.ServiceModel.Web.dll |
+| /microsoft.netcore.app.ref/6.0.13/ref/net6.0/System.ServiceProcess.dll |
+| /microsoft.netcore.app.ref/6.0.13/ref/net6.0/System.Text.Encoding.CodePages.dll |
+| /microsoft.netcore.app.ref/6.0.13/ref/net6.0/System.Text.Encoding.Extensions.dll |
+| /microsoft.netcore.app.ref/6.0.13/ref/net6.0/System.Text.Encoding.dll |
+| /microsoft.netcore.app.ref/6.0.13/ref/net6.0/System.Text.Encodings.Web.dll |
+| /microsoft.netcore.app.ref/6.0.13/ref/net6.0/System.Text.Json.dll |
+| /microsoft.netcore.app.ref/6.0.13/ref/net6.0/System.Text.RegularExpressions.dll |
+| /microsoft.netcore.app.ref/6.0.13/ref/net6.0/System.Threading.Channels.dll |
+| /microsoft.netcore.app.ref/6.0.13/ref/net6.0/System.Threading.Overlapped.dll |
+| /microsoft.netcore.app.ref/6.0.13/ref/net6.0/System.Threading.Tasks.Dataflow.dll |
+| /microsoft.netcore.app.ref/6.0.13/ref/net6.0/System.Threading.Tasks.Extensions.dll |
+| /microsoft.netcore.app.ref/6.0.13/ref/net6.0/System.Threading.Tasks.Parallel.dll |
+| /microsoft.netcore.app.ref/6.0.13/ref/net6.0/System.Threading.Tasks.dll |
+| /microsoft.netcore.app.ref/6.0.13/ref/net6.0/System.Threading.Thread.dll |
+| /microsoft.netcore.app.ref/6.0.13/ref/net6.0/System.Threading.ThreadPool.dll |
+| /microsoft.netcore.app.ref/6.0.13/ref/net6.0/System.Threading.Timer.dll |
+| /microsoft.netcore.app.ref/6.0.13/ref/net6.0/System.Threading.dll |
+| /microsoft.netcore.app.ref/6.0.13/ref/net6.0/System.Transactions.Local.dll |
+| /microsoft.netcore.app.ref/6.0.13/ref/net6.0/System.Transactions.dll |
+| /microsoft.netcore.app.ref/6.0.13/ref/net6.0/System.ValueTuple.dll |
+| /microsoft.netcore.app.ref/6.0.13/ref/net6.0/System.Web.HttpUtility.dll |
+| /microsoft.netcore.app.ref/6.0.13/ref/net6.0/System.Web.dll |
+| /microsoft.netcore.app.ref/6.0.13/ref/net6.0/System.Windows.dll |
+| /microsoft.netcore.app.ref/6.0.13/ref/net6.0/System.Xml.Linq.dll |
+| /microsoft.netcore.app.ref/6.0.13/ref/net6.0/System.Xml.ReaderWriter.dll |
+| /microsoft.netcore.app.ref/6.0.13/ref/net6.0/System.Xml.Serialization.dll |
+| /microsoft.netcore.app.ref/6.0.13/ref/net6.0/System.Xml.XDocument.dll |
+| /microsoft.netcore.app.ref/6.0.13/ref/net6.0/System.Xml.XPath.XDocument.dll |
+| /microsoft.netcore.app.ref/6.0.13/ref/net6.0/System.Xml.XPath.dll |
+| /microsoft.netcore.app.ref/6.0.13/ref/net6.0/System.Xml.XmlDocument.dll |
+| /microsoft.netcore.app.ref/6.0.13/ref/net6.0/System.Xml.XmlSerializer.dll |
+| /microsoft.netcore.app.ref/6.0.13/ref/net6.0/System.Xml.dll |
+| /microsoft.netcore.app.ref/6.0.13/ref/net6.0/System.dll |
+| /microsoft.netcore.app.ref/6.0.13/ref/net6.0/WindowsBase.dll |
+| /microsoft.netcore.app.ref/6.0.13/ref/net6.0/mscorlib.dll |
+| /microsoft.netcore.app.ref/6.0.13/ref/net6.0/netstandard.dll |
+| /netstandard.library/2.0.3/build/netstandard2.0/ref/System.ComponentModel.Composition.dll |
+| /newtonsoft.json/12.0.1/lib/net45/Newtonsoft.Json.dll |
```

Comment:
There are multiple `Newtonsoft.Json.dll` variants in this Nuget package. They have the same name and version. Our logic for choosing one of them is not stable, so it's not working for integration test purposes.